### PR TITLE
Adjust messaging examples to match v1.0.0 API

### PR DESCRIPTION
**Pull request by madebynoxc**
**Select the type of change:**

- [x] Small Changes - Typos, formatting, slight revisions
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site and Docusaurus

**Purpose of the Pull Request:**
Adjust `CustomMessagingManager` examples to match v1.0.0 API as it is now part of the `NetworkManager`

### CODE REVIEW

**Comment by unity-cla-assistant** [comment]
[CLA assistant check](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=387) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.cds.internal.unity3d.com/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=387) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.cds.internal.unity3d.com/check/Unity-Technologies/com.unity.multiplayer.docs?pullRequest=387) it.</sub>

**Comment by s-omeilia-unity** [comment]
@TwoTenPvP I'm cleaning up all the open Docs PRs. Is this content accurate for me to merge?

File: versioned_docs/version-1.0.0/advanced-topics/message-system/custom-messages.md [hunk 1]
```diff
@@ -45,14 +45,14 @@ If you want a completed messaging system, you can use named messages. The receiv
 private void Start()
 {
     //Receiving
-    CustomMessagingManager.RegisterNamedMessageHandler("myMessageName", (senderClientId, reader) =>
+    NetworkManager.CustomMessagingManager.RegisterNamedMessageHandler("myMessageName", (senderClientId, reader) =>
```
Comment (TwoTenPvP) [inline versioned_docs/version-1.0.0/advanced-topics/message-system/custom-messages.md:48]:
```suggestion
    NetworkManager.Singleton.CustomMessagingManager.RegisterNamedMessageHandler("myMessageName", (senderClientId, reader) =>
```

File: versioned_docs/version-1.0.0/advanced-topics/message-system/custom-messages.md [hunk 2]
```diff
@@ -45,14 +45,14 @@ If you want a completed messaging system, you can use named messages. The receiv
...
     {
         reader.ReadValueSafe(out string message); //Example
     });
 
     //Sending
     using FastBufferWriter writer = new FastBufferWriter(maxSize, Allocator.Temp);
     writer.WriteValueSafe(message);
-    CustomMessagingManager.SendNamedMessage("myMessageName", clientId, writer, NetworkDelivery.Reliable); //NetworkDelivery is optional.
+    NetworkManager.CustomMessagingManager.SendNamedMessage("myMessageName", clientId, writer, NetworkDelivery.Reliable); //NetworkDelivery is optional.
```
Comment (TwoTenPvP) [inline versioned_docs/version-1.0.0/advanced-topics/message-system/custom-messages.md:56]:
```suggestion
    NetworkManager.Singleton.CustomMessagingManager.SendNamedMessage("myMessageName", clientId, writer, NetworkDelivery.Reliable); //NetworkDelivery is optional.
```

File: versioned_docs/version-1.0.0/advanced-topics/message-system/custom-messages.md [hunk 3]
```diff
@@ -22,15 +22,15 @@ Unnamed messages can be thought of as a single sending channel. A message sent h
...
+    NetworkManager.CustomMessagingManager.OnUnnamedMessage += ((senderClientId, reader) =>
     {
         reader.ReadValueSafe(out string message); //Example
     });
 
     //Sending
     using FastBufferWriter writer = new FastBufferWriter(maxSize, Allocator.Temp);
     writer.WriteValueSafe(message);
-    CustomMessagingManager.SendUnnamedMessage(clientId, writer, NetworkDelivery.Reliable); //NetworkDelivery is optional.
+    NetworkManager.CustomMessagingManager.SendUnnamedMessage(clientId, writer, NetworkDelivery.Reliable); //NetworkDelivery is optional.
```
Comment (TwoTenPvP) [inline versioned_docs/version-1.0.0/advanced-topics/message-system/custom-messages.md:33]:
```suggestion
    NetworkManager.Singleton.CustomMessagingManager.SendUnnamedMessage(clientId, writer, NetworkDelivery.Reliable); //NetworkDelivery is optional.
```

File: versioned_docs/version-1.0.0/advanced-topics/message-system/custom-messages.md [hunk 4]
```diff
@@ -22,15 +22,15 @@ Unnamed messages can be thought of as a single sending channel. A message sent h
 private void Start()
 {
     //Receiving
-    CustomMessagingManager.OnUnnamedMessage += ((senderClientId, reader) =>
+    NetworkManager.CustomMessagingManager.OnUnnamedMessage += ((senderClientId, reader) =>
```
Comment (TwoTenPvP) [inline versioned_docs/version-1.0.0/advanced-topics/message-system/custom-messages.md:25]:
```suggestion
    NetworkManager.Singleton.CustomMessagingManager.OnUnnamedMessage += ((senderClientId, reader) =>
```

**Review by TwoTenPvP — CHANGES_REQUESTED** [review]
With these changes, the content is accurate and should be merged :)

**Comment by s-omeilia-unity** [comment]
Closing this PR because all of the API refs are being rebuilt from scratch with the most up-to-date content for the 1.0.0 full release.

**Pull request closed without merging**